GOVERNANCE.md

```diff
@@ -68,9 +68,23 @@ Members participate in a vote.
 
 ## Merging PRs into this Repository
 
-This section does not apply to [the Node.js core repository](https://github.com/nodejs/node).
-It only applies to [the bootstrap repository](https://github.com/nodejs/bootstrap).
+Pull requests that do not change the charter or governance of the CPC can be merged into
+this repository provided the following conditions have been met:
 
-Pull requests must have no objections and have been open for at least 72 hours 
-in order to be merged into this repository. A pull request that is unable to
-reach consensus cannot be merged into this repository.
+* there are no outstanding objections
+* there are two approvals by CPC members
```

Comment:
is this Regular Members? Voting Members? Observers? Any member of any project?

Reply:
Any member, regular and voting